spec/factories/vehicle_factory.rb

```diff
@@ -0,0 +1,9 @@
+FactoryBot.define do
+  factory :vehicle do
+    assessment { create :assessment }
```

Comment:
You can just write `assessment` here.
Like on this other factory: https://github.com/ministryofjustice/check-financial-eligibility/blob/master/spec/factories/property_factory.rb#L3
```suggestion
    assessment
```